### PR TITLE
docs: remove "new" from media queries

### PR DESCRIPTION
**Pull request by thaytharma**
Remove "new" status from media queries

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 title: 'Media Queries'
-status: 'new'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/usage/layout/media-queries.md:3]: 9 months old, I agree that it's not new anymore 🙏

**Review by langz — APPROVED** [review]
🆕

**Review by tujoworker — APPROVED** [review]
👍

**Pull request merged**